services/frontend-service/pkg/cmd/server.go

```diff
@@ -125,14 +125,21 @@ func RunServer() {
 	}
 }
 
+func stripSecrets(config config.ServerConfig) config.ServerConfig {
```

Comment:
I do not pass by reference so I do not change the `config.ServerConfig` object itself, as we may need the `real` value of `config.ServerConfig.DexClientSecret` later on and only want to hide it in the logs